.github/workflows/validate-nightly-binaries.yml

```diff
@@ -0,0 +1,27 @@
+# Scheduled validation of the nightly binaries
+name: cron
+
+on:
+  schedule:
+    # At 5:30 pm UTC (7:30 am PDT)
+    - cron: "30 17 * * *"
+  # Have the ability to trigger this job manually through the API
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/validate-nightly-binaries.yml
+      - .github/workflows/validate-binaries.yml
+      - .github/scripts/validate-binaries.sh
+  pull_request:
+    paths:
+      - .github/workflows/validate-nightly-binaries.yml
+      - .github/workflows/validate-binaries.yml
+      - .github/scripts/validate-binaries.sh
+jobs:
+  nightly:
+    uses: ./.github/workflows/validate-binaries.yml
+    with:
+      channel: nightly
```

Comment:
so, this will only validate nightly now, but the validate-binaries.yml will have the ability to validate all channels?

Reply:
Not necessary. Nightly job will validate nightly only. To validate release this PR adds validate-binaries workflow and workflow_dispatch section enabled you to run this on release channel. You can see it for example in torchtext:
https://github.com/pytorch/text/actions/workflows/validate-binaries.yml